src/content/blog/ddev-jan-2025-newsletter.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "DDEV January 2025 Newsletter"
+pubDate: 2025-01-08
+#modifiedDate: 2024-09-06
+summary: DDEV January 2025 Newsletter
+author: Randy Fay
+#featureImage:
+#  src: /img/blog/2024/12/nancy_lewis_colorado_river.jpg
+#  alt: Nancy Lewis painting of Colorado River near Palisade, Colorado
+#  credit: Nancy Lewis painting of the Colorado River near Palisade, Colorado
+categories:
+  - Community
+---
+
+**Happy New Year DDEV Community!**
+
+Our most important news of the month is both bad and good. Our lead sponsor, Platform.sh, has decided to reduce and change its sponsorship amount, but also to transfer to the DDEV Foundation the "DDEV" trademark and the domains that go with it, and they will continue sponsoring DDEV at a lower financial level. You can read all about it in the [change announcement blog](20241226_platformsh_changes20241226_platformsh_changes). But the TL;DR is that we need every organization and individual to step up their financial support in order to ensure DDEV's future.
+
+**DDEV v1.24.1** was released with minor changes.
+
+**DDEV's TYPO3 Community Budget Idea was accepted!** TYPO3's [community budget process](https://typo3.org/article/budget-2025-ideas-for-quarter-1-2025-published-vote-now) is an outstanding process for funding community ideas. We'll be improving DDEV's xhprof and xhgui even beyond where [@tyler36](https://github.com/tyler36) has already taken the [DDEV xhgui add-on](https://github.com/ddev/ddev-xhgui) and incorporate it into core, and at completion the TYPO3 Association will fund the project for €7500, a fantastic contribution.
+
+**DDEV Notes and News**
+
+- [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). This month's meeting is Wednesday, January 8, 2025, and is a critical discussion of our financial situation, budget, and plans based on recent news [join us](https://www.meetup.com/ddev-events/events/303197425). If you'd like a calendar invitation instead of the meetup signup, just reply to this email.
+- [DDEV, solr, and platform.sh](https://www.computerminds.co.uk/articles/ddev-solr-and-platformsh)
+- [Joshuami blog on Drupal CMS (via DDEV)](https://joshuami.com/blog/2024-12/blog-drupal-cms-starshot-kinda-now/)
+- [Testing Drupal 11 with Cypress and DDEV](https://www.thedroptimes.com/44845/how-test-drupal-11-using-cypress-and-ddev-drupaladicto-explains) shows how to use [@tyler36](https://github.com/tyler36)'s excellent [Cypress add-on for DDEV](https://github.com/tyler36/ddev-cypress).
```

Comment:
It links to a Spanish-lanaguage setup video.

Reply:
I hadn't realized that. Glad I didn't lik directly to the video.